Can't run initialization hooks in the JVM image

I'm trying to run a initialization script to create a DynamoDB table and discovered that the jvm images don't seem to include bash, so they fail to start when init scripts are present:
```
2026-04-06 14:02:37,569 INFO  [io.quarkus] (main) floci 1.3.0 on JVM (powered by Quarkus 3.32.3) started in 0.768s. Listening on: http://0.0.0.0:4566
2026-04-06 14:02:37,569 INFO  [io.quarkus] (main) Profile prod activated.
2026-04-06 14:02:37,569 INFO  [io.quarkus] (main) Installed features: [cdi, config-yaml, rest, rest-jackson, smallrye-context-propagation, vertx]
2026-04-06 14:02:37,585 INFO  [io.github.hectorvent.floci.lifecycle.inithook.InitializationHooksRunner] (init-hooks-runner) Running startup hook with 1 script(s) from /etc/floci/init/start.d: [init-table.sh]
2026-04-06 14:02:37,591 INFO  [io.github.hectorvent.floci.lifecycle.inithook.InitializationHooksRunner] (init-hooks-runner) Executing startup hook script: init-table.sh
2026-04-06 14:02:37,592 ERROR [io.github.hectorvent.floci.lifecycle.EmulatorLifecycle] (init-hooks-runner) Startup hook execution failed — shutting down: java.io.IOException: Cannot run program "/bin/bash" (in directory "/etc/floci/init/start.d"): Exec failed, error: 2 (No such file or directory)
	at java.base/java.lang.ProcessBuilder.start(Unknown Source)
	at java.base/java.lang.ProcessBuilder.start(Unknown Source)
	at io.github.hectorvent.floci.lifecycle.inithook.HookScriptExecutor.run(HookScriptExecutor.java:28)
	at io.github.hectorvent.floci.lifecycle.inithook.HookScriptExecutor_ClientProxy.run(Unknown Source)
	at io.github.hectorvent.floci.lifecycle.inithook.InitializationHooksRunner.run(InitializationHooksRunner.java:68)
	at io.github.hectorvent.floci.lifecycle.inithook.InitializationHooksRunner.run(InitializationHooksRunner.java:54)
	at io.github.hectorvent.floci.lifecycle.inithook.InitializationHooksRunner_ClientProxy.run(Unknown Source)
	at io.github.hectorvent.floci.lifecycle.EmulatorLifecycle.runStartupHooksAfterReady(EmulatorLifecycle.java:69)
	at java.base/java.lang.VirtualThread.run(Unknown Source)
Caused by: java.io.IOException: Exec failed, error: 2 (No such file or directory)
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(Unknown Source)
	at java.base/java.lang.ProcessImpl.start(Unknown Source)
	... 9 more

2026-04-06 14:02:37,599 INFO  [io.github.hectorvent.floci.lifecycle.EmulatorLifecycle] (main) === AWS Local Emulator Shutting Down ===
2026-04-06 14:02:37,600 INFO  [io.github.hectorvent.floci.services.elasticache.proxy.ElastiCacheProxyManager] (main) Stopped all ElastiCache proxies
2026-04-06 14:02:37,600 INFO  [io.github.hectorvent.floci.services.rds.proxy.RdsProxyManager] (main) Stopped all RDS proxies
2026-04-06 14:02:37,600 INFO  [io.github.hectorvent.floci.lifecycle.EmulatorLifecycle] (main) === AWS Local Emulator Stopped ===
```
Note these fail even if the init script itself doesn't use bash.

--- a/docs/configuration/application-yml.md
+++ b/docs/configuration/application-yml.md
@@ -42,7 +42,7 @@ floci:
     presign-secret: local-emulator-secret  # HMAC secret for S3 pre-signed URL verification
 
   init-hooks:
-    shell-executable: /bin/bash
+    shell-executable: /bin/sh
     timeout-seconds: 30
     shutdown-grace-period-seconds: 2
 

--- a/docs/configuration/initialization-hooks.md
+++ b/docs/configuration/initialization-hooks.md
@@ -117,7 +117,7 @@ You can customize hook behavior via configuration:
 
 | Key                                              | Default     | Description                                                                                                      |
 |--------------------------------------------------|-------------|------------------------------------------------------------------------------------------------------------------|
-| `floci.init-hooks.shell-executable`              | `/bin/bash` | Shell executable used to run scripts                                                                             |
+| `floci.init-hooks.shell-executable`              | `/bin/sh`   | Shell executable used to run scripts                                                                             |
 | `floci.init-hooks.timeout-seconds`               | `30`        | Maximum execution time per script before it is terminated and considered failed                                  |
 | `floci.init-hooks.shutdown-grace-period-seconds` | `2`         | Time to wait after calling `destroy()` before forcefully stopping the process (allows cleanup hooks to complete) |
 

--- a/src/main/java/io/github/hectorvent/floci/config/EmulatorConfig.java
+++ b/src/main/java/io/github/hectorvent/floci/config/EmulatorConfig.java
@@ -430,7 +430,7 @@ public interface EmulatorConfig {
     }
 
     interface InitHooksConfig {
-        @WithDefault("/bin/bash")
+        @WithDefault("/bin/sh")
         String shellExecutable();
 
         @WithDefault("2")

--- a/src/test/resources/application.yml
+++ b/src/test/resources/application.yml
@@ -37,7 +37,7 @@ floci:
     presign-secret: test-secret
 
   init-hooks:
-    shell-executable: /bin/bash
+    shell-executable: /bin/sh
     timeout-seconds: 5
     shutdown-grace-period-seconds: 2
 

